src/main/resources/static/config.soy

```diff
@@ -21,4 +21,12 @@
         {param errorTexts: $errors ? $errors['jira-version-prefix'] : null /}
         {param isRequired: false /}
     {/call}
+    {call aui.form.textField}
+        {param id: 'release-commit-version-pattern' /}
+        {param value: $config['release-commit-version-pattern'] /}
+        {param labelContent: 'Release commit version pattern' /}
+        {param descriptionText: 'Pattern that matches the release commit message. Must contain named capturing group with a name "version". Default pattern is "\\[maven-release-plugin\\] prepare release repositoryName-(?<version>.*)" where repositoryName is the name of the stash repository.' /}
```

Comment:
The current soy template results in https://github.com/infobip/jira-version-generator/blob/issue-%232-version-commit-pattern/docs/release-commit-version-pattern.png.
This was achieved through trial and error since I couldn't find any documenation on the case. :)